package/src/components/Message/Message.tsx

```diff
@@ -312,9 +312,12 @@ const MessageWithContext = <
     !isMessageTypeDeleted && Array.isArray(message.attachments)
       ? message.attachments.reduce(
           (acc, cur) => {
-            if (cur.type === 'file' || cur.type === 'video') {
+            if (cur.type === 'file') {
               acc.files.push(cur);
               acc.other = []; // remove other attachments if a file exists
+            } else if (cur.type === 'video' && !cur.og_scrape_url) {
+              acc.images.push({ image_url: cur.asset_url, type: 'video' });
+              acc.other = [];
```

Comment:
@Enigma-I-am This change actually fixes the issue around url preview for youtube links.